input/provision_config.yml

```diff
@@ -36,24 +38,38 @@ admin_nic_subnet: "172.29.0.0"
 pxe_nic: "eno1"
 
 # The dhcp range for assigning the IPv4 address while provisioning
+# Examples: 172.29.0.50 - 172.29.0.100, 10.10.1.100 - 10.10.1.200
+# Default: 172.29.0.100 - 172.29.0.200
 # Mandatory
 pxe_nic_start_range: "172.29.0.100"
 pxe_nic_end_range: "172.29.0.200"
 
 ### IB NIC Details
 # If provided, omnia will handle and assign static IPs to compute node's IB network.
+# Only last 16 bits/2 octets of IPv4 are dynamic
+# Example: 172.29.0.0 , 10.10.0.0 , 192.15.0.0
```

Comment:
Can we have it different subnet here like this:
Examples: 172.27.0.0, 10.8.0.0, 192.13.0.0, note second octet of the subnet is different.